src/app/query/bitcoin/ordinals/use-inscriptions.query.ts

```diff
@@ -149,9 +171,6 @@ export function useInscriptionsInfiniteQuery() {
 
       return { offset: calculatedOffset, total };
     },
-    refetchOnMount: false,
```

Comment:
wonder if it really should be removed? wdyt @kyranjamie

Reply:
I would add back. As this is recursive, we don't want it happening too often.

Reply:
added stale time as well